Documentation/dev/libvirt-howto.md

```diff
@@ -85,6 +85,35 @@ iptables -I INPUT -p tcp -s 192.168.124.0/24 -d 192.168.124.1 --dport 16509 \
   -j ACCEPT -m comment --comment "Allow insecure libvirt clients"
 ```
 
+If using `firewalld`, simply optain the name of the existing active zone which 
+can be used to integrate the appropriate source and ports to allow connections from
+the IP range used by your cluster nodes. An example is shown below.
+
+````
+sudo firewall-cmd --get-active-zones
+FedoraWorkstation
+  interfaces: enp0s25 tun0
+````
+With the name of the active zone, include the source and port to allow connections
+from the IP range used by your cluster nodes. The default subnet is `192.168.124.0/24`
+unless otherwise specified. 
+
+````
```

Comment:
You don't have any output here, and for shell examples without output we prefer no prompts and:

    ```sh

syntax highlighting (more on this in #27).